httpretty/core.py

```diff
@@ -703,6 +703,10 @@ def __getattr__(self, name):
                 # calls (or can they?)
                 self.truesock = self.create_socket()
             elif not self.truesock:
+                # Special case for
+                # `hasattr(sock, "version")` call added in urllib3>=1.26.
+                if name == 'version':
+                    raise AttributeError()
```

Comment:
Thanks again for the fix, but please add a user-friendly message to help future HTTPretty users understand this rather esoteric issue.

Here is a stereotypical example:

```suggestion
                    raise AttributeError("HTTPretty synthesized this error to fix urllib3 compatibility (see issue https://github.com/gabrielfalcao/HTTPretty/issues/409). Please open an issue if this error causes further unexpected issues.")
```

Reply:
Good idea. I'll give this a try and make sure it plays nice with `@property`'s `AttributeError` error handling.

Reply:
Message added!